<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="3" passed="2">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="6166" started-at="2021-02-08T19:42:02 CET" finished-at="2021-02-08T19:42:08 CET">
    <groups>
    </groups>
    <test name="Default test" duration-ms="6166" started-at="2021-02-08T19:42:02 CET" finished-at="2021-02-08T19:42:08 CET">
      <class name="tests.TestSauceDemo">
        <test-method status="PASS" signature="createDriver()[pri:0, instance:tests.TestSauceDemo@23faf8f2]" name="createDriver" is-config="true" duration-ms="2949" started-at="2021-02-08T19:42:02 CET" finished-at="2021-02-08T19:42:05 CET">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- createDriver -->
        <test-method status="FAIL" signature="testLoginWrong()[pri:1, instance:tests.TestSauceDemo@23faf8f2]" name="testLoginWrong" duration-ms="1791" started-at="2021-02-08T19:42:05 CET" finished-at="2021-02-08T19:42:07 CET">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[nevalidan expected [https://www.saucedemo.com/inventory.html] but found [https://www.saucedemo.com/]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: nevalidan expected [https://www.saucedemo.com/inventory.html] but found [https://www.saucedemo.com/]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at tests.TestSauceDemo.testLoginWrong(TestSauceDemo.java:61)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testLoginWrong -->
        <test-method status="PASS" signature="testLoginCorrect()[pri:2, instance:tests.TestSauceDemo@23faf8f2]" name="testLoginCorrect" duration-ms="950" started-at="2021-02-08T19:42:07 CET" finished-at="2021-02-08T19:42:08 CET">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testLoginCorrect -->
        <test-method status="PASS" signature="sortTest()[pri:3, instance:tests.TestSauceDemo@23faf8f2]" name="sortTest" duration-ms="386" started-at="2021-02-08T19:42:08 CET" finished-at="2021-02-08T19:42:08 CET">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- sortTest -->
      </class> <!-- tests.TestSauceDemo -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
